### DISCUSSION TITLE
v0.5.6

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#272] [New Releases]
## What's Changed
* Add `Alt+Down` to duplicate current line
* Add shortcut for Replace
* Add ability to detect language from start of file
* Auto detect XML files
* Handle multi-byte characters in regex matching
* Highlight and enable `Ctrl+Click` to open URLs located in the file
* Show hidden files on file dialogs for Linux
* Save/Restore editor zoom level
* Update Chinese translation
* Optimize application closing with opened files
* Fix Javascript syntax highlighting
* Fix displaying of non-ascii chars in Hex Viewer
* Fix Scintilla's RectangularSelectionModifier for Qt
* Fix forcing window to foreground on Windows
* Attempt to bring window to foreground on Linux
* Update Scintilla to v5.3.1
* Update Lexilla to v5.2.0

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.5...v0.5.6

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.5.6'>v0.5.6</a>.</em>

**Comment by sxd9602** [comment]
If you press  "shift + _"  at the same time, the system will blink back

**Reply by dail8859** [reply]
What does "system will blink" even mean? You'll have to provide more details (OS, distribution, etc) and also a screenshot or video since I can't even begin to guess what the problem is.

**Reply by sxd9602** [reply]
On macos, when you press shift + _, the software automatically exits

**Reply by sxd9602** [reply]
Can you speak Chinese？
在macos系统上面，同时按下 "shift + _"按键，无论哪个版本的Notepadnext都会出现闪退的情况。

**Reply by dail8859** [reply]
> Can you speak Chinese?

I cannot.

> On the macos system, press the "shift + _" button at the same time, no matter which version of Notepadnext will crash.

Is this some kind of special shortcut for macOS? I don't know why this shortcut would crash the application and other shortcuts would not.

**Reply by sxd9602** [reply]
> > Can you speak Chinese?
> 
> I cannot.
> 
> > On the macos system, press the "shift + _" button at the same time, no matter which version of Notepadnext will crash.
> 
> Is this some kind of special shortcut for macOS? I don't know why this shortcut would crash the application and other shortcuts would not.

Should not be the problem of shortcut, Chinese input method, while pressing "shift + _" will cause the program crash, English input method, the program runs normally

**Reply by dail8859** [reply]
> Should not be the problem of shortcut, Chinese input method, while pressing "shift + _" will cause the program crash, English input method, the program runs normally

See #194

**Comment by sxd9602** [comment]
If you press shift + _ at the same time, the software will blink
